src/auth.ts

```diff
@@ -165,16 +167,18 @@ export async function authorize(options: {
         },
         isLocalCreds: true,
       };
-      await DOTFILE.RC_LOCAL().write(claspToken);
+      // await DOTFILE.RC_LOCAL().write(claspToken);
     } else {
+      dotfile = DOTFILE.RC;
       // Save global ClaspCredentials.
       claspToken = {
         token,
         oauth2ClientSettings: globalOauth2ClientSettings,
         isLocalCreds: false,
       };
-      await DOTFILE.RC.write(claspToken);
+      // await DOTFILE.RC.write(claspToken);
```

Comment:
Why is this commented? Can we remove it?

Reply:
gone